doc/source/working_with_kiwi.rst

```diff
@@ -84,3 +85,149 @@ to find in the *description directory*:
           <archive name="custom-archive.tgz"/>
       </packages>
 
+
+.. _working-with-kiwi-image-building-process:
+
+Image Building Process
+----------------------
+
+KIWI creates images in a two step process. The first step, the *prepare*
+operation, generates a so-called *unpacked image* tree (directory) using
+the information provided in the :file:`config.xml` configuration file
+(see :ref:`xml-description`)
+
+The second step, the *create* operation, creates the *packed image* or
+*image* in the specified format based on the unpacked image and the
+information provided in the :file:`config.xml` configuration file.
+
+.. figure:: .images/intro.png
+    :align: center
+    :alt: Image Creation Architecture
+
+    Image Creation Architecture
+
+
+.. _prepare-step:
+
+The Prepare Step
+^^^^^^^^^^^^^^^^
+
+The first step of the image creation is the *prepare* step, during which
+KIWI creates an *unpacked image*, also called the "root tree". This
+directory will be the installation target for software packages to be
+installed during the image creation process.
+
+For the package installation, KIWI relies on the package manager specified
+in the ``packagemanager`` element in :file:`config.xml` (see
+:ref:`xml-description-preferences-common-elements`). KIWI supports the
+following package managers: ``dnf``, ``zypper`` (default) and ``apt-get``.
+
+The prepare step consists of the following substeps:
+
+#. **Create Target Root Directory.**
+
+   KIWI will exit with an error if the target root tree already exists to
+   avoid accidental deletion of an existing unpacked image.
+
+#. **Install Packages.**
+
+   KIWI first configures the package manager to use the repositories
+   specified in the configuration file and/or via the command line. After
+   the repository setup, the packages specified in the ``bootstrap``
+   section of the image description are installed in a temporary workspace
+   external to the target root tree. This establishes the initial
+   environment to support the completion of the process in a chroot
+   setting. The essential packages to specify as part of the bootstrap
```

Comment:
The following sentence is a bit weird to me. I've reordered it a bit, hope I didn't change the meaning:

```suggestion
   setting. The essential packages are ``filesystem`` and ``glibc-locale`` to specify
```